tests/src/test/java/alluxio/client/cli/fs/command/CopyFromLocalCommandIntegrationTest.java

```diff
@@ -306,6 +306,12 @@ public void copyFromLocalWildcard() throws IOException {
 
   @Test
   public void copyFromLocalRelativePath() throws Exception {
+    String version = System.getProperty("java.version");
```

Comment:
could we do  `CommonUtil::getJavaVersion() >= 11` instead? so that we can avoid changing this when we go to java 14, whenever that is.

Reply:
I would like to remove or replace this test when we get there.